src/plugins/console/public/lib/autocomplete/autocomplete.ts

```diff
@@ -357,6 +398,12 @@ export default function ({
   }) {
     const context = term.context!;
 
+    // If term contains conditional definitions, it returns the correct template based on the value configured in the request.
+    const template = getConditionalTemplate(term);
+    if (template) {
+      term.template = template as any;
```

Comment:
Why `any` is here? We must avoid `any` type. If you don't know the type, it is safer to add `unknown` type or real type, if it is possible.